awesome_owl/static/src/todo_list/todo_list.js

```diff
@@ -0,0 +1,28 @@
+import { Component, useState } from "@odoo/owl";
+import { useAutofocus } from "../utils";
+import { Todo } from "./todo";
+import { TodoItem } from "./todo_item";
+
+export class TodoList extends Component {
+    static template = "awesome_owl.todo_list";
+    static components = { TodoItem };
+
+    setup() {
+        this.todos = useState([]);
+        useAutofocus("input");
+    }
+
+    addTodo(ev) {
+        if (ev.keyCode === 13 && ev.target.value != "") {
```

Comment:
nitpick (i know the tutorial makes you do it):
```suggestion
        if (ev.key === "Enter" && ev.target.value) {
```